src/main.js

```diff
@@ -40,11 +40,23 @@ Vue.filter("formatUnit", unit => {
 
 //transforms sats to usd
 Vue.filter("satsToUSD", value => {
```

Comment:
```suggestion
//transforms sats to fiat
Vue.filter("satsToFiat", value => {
```
More logical, but applying this suggestion means change all the "satsToUSD" function calls to "satsToFiat".

Reply:
Agree with all this. 
I'll make some changes. 
I went back and forth with the naming / wording.